home/tests.py

```diff
@@ -0,0 +1,17 @@
+from django.test import TestCase, Client
+from mock import patch, Mock
+
+
+class HomeTestCase(TestCase):
+    def test_when_home_visited_then_environment_variable_is_valid(self):
+        settings_mocked = self.create_mock('home.views.settings')
+        settings_mocked.ENV_NAME = 'Test'
```

Comment:
The tests are not passing, I think it is because `ENV_NAME` should be `ENVIRONMENT_NAME`